ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>Limit are at 0.5 deg at this moment right? Lets change movement to +-5 deg as suggested on page 22.</p><p> </p><p>For the next step send annotations to the EFD and mention that the next step is done by the EUI</p><ul><li>Execute a 3.5 deg/s constant speed move. The rotator should stop 1 s after hitting the 0.5 deg positive limit</li><li>Execute a 3.5 deg/s constant speed move. The rotator should not move.</li></ul><p> </p><p>Step done by the EUI should be included in the plots.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>This test condition is documented in the TMA test plan. If you would like to change the conditions, please contact with Ander in Tekniker to update the test plan document. Thanks!</p><p> </p>